noisecapture_calibration.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+     <head>
+
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Noise-Planet - Scientific tools for environmental noise assessment">
+        <meta name="author" content="Gwendall Petit (Lab-STICC - CNRS UMR 6285 / DECIDE Team)">
```

Comment:
Lab-STICC - CNRS UMR 6285 / DECIDE Team must be CNRS , Lab-STICC UMR 6285 / DECIDE Team